http.go

```diff
@@ -96,8 +96,17 @@ type lookupResponse struct {
 }
 
 func handleLookup(items []apps.SelectOption) callHandler {
-	return func(_ *apps.CallRequest) apps.CallResponse {
-		return apps.NewDataResponse(lookupResponse{items})
+	return func(creq *apps.CallRequest) apps.CallResponse {
+		query := strings.ToLower(creq.Query)
```

Comment:
This makes it so the lookup calls have the text filtered by matching the user's input text. You'll need to deploy this PR of the apps plugin to use this feature https://github.com/mattermost/mattermost-plugin-apps/pull/304